client/app/scripts/superdesk-desks/views/aggregate-widget.html

```diff
@@ -1,7 +1,10 @@
 <div class="content-search" ng-class="{extended: agg.selected}" ng-controller="AggregateWidgetCtrl as agg">
     <div class="search-container">
         <div class="search-box">
-            <input type="text" placeholder="{{ :: 'Search'|translate }}" ng-model="query">
+            <input type="text" placeholder="{{ :: 'Search' | translate }}"
+                ng-model="query"
+                ng-model-options="{debounce: 500}"

```

Comment:
Glad to see that `ng-model-options` are becoming more common in our code. :wink: